app/templates/index-g-cloud.html

```diff
@@ -25,7 +25,7 @@
   </div>
   <ul class="browse-list">
     <li class="browse-list-item">
-      <a href="/search?q=&lot=saas" class="browse-list-item-link">
+      <a href="/search/?q=&lot=saas" class="browse-list-item-link">
```

Comment:
It is at the moment, I've added this as a bug: https://www.pivotaltracker.com/story/show/94931176